doc/whats_new.rst

```diff
@@ -583,6 +583,11 @@ Preprocessing and feature selection
       of ``X`` to transform function when ``copy=True`` (:issue:`7194`). By `Caio
       Oliveira <https://github.com/caioaao>`_.
 
+    - For sparse matrices, :func:`preprocessing.normalize with ``return_norm=True``
```

Comment:
You need to put this entry in the 0.19 section, i.e. the version in development (currently it is in the 0.18). I would put that in Enhancements rather than Bug Fixes since the impact of your PR is a better error message.